src/panels/config/lovelace/ha-config-lovelace.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  HassRouterPage,
+  RouterOptions,
+} from "../../../layouts/hass-router-page";
+import { property, customElement } from "lit-element";
+import { HomeAssistant } from "../../../types";
+
+export const lovelaceTabs = [
+  {
+    component: "lovelace",
+    path: "/config/lovelace/dashboards",
+    translationKey: "ui.panel.config.lovelace.dashboards.caption",
+    icon: "hass:view-dashboard",
+  },
+  {
+    component: "lovelace",
+    path: "/config/lovelace/resources",
+    translationKey: "ui.panel.config.lovelace.resources.caption",
+    icon: "hass:file-multiple",
+    expertOnly: true,
```

Comment:
Is it going to look weird if we have just 1 tab? Should we hide the tabs at the bottom if we do?